packages/@react-spectrum/dnd/src/useDnDHooks.ts

```diff
@@ -41,8 +41,10 @@ import {
 import {DroppableCollectionProps} from '@react-types/shared';
 import {Key, RefObject, useMemo} from 'react';
 
+interface DraggableCollectionStateOpts extends Omit<DraggableCollectionStateOptions, 'getItems'> {}
+
 interface DragHooks {
-  useDraggableCollectionState?: (props: Omit<DraggableCollectionStateOptions, 'getItems'>) => DraggableCollectionState,
```

Comment:
The type expanded weird in the docs so had to move to separate interface

Reply:
If it isn't omitted, calls of `dndHook.useDraggableCollectionState` (like in ListView [here](https://github.com/adobe/react-spectrum/blob/4def693b76deb25342ef818750f5698b70718d0a/packages/%40react-spectrum/list/src/ListView.tsx#L146-L151)) will run into a typescript error complaining that `getItems` isn't provided even though it is provided by the `useDnDHooks` options [here](https://github.com/adobe/react-spectrum/blob/bb120eacb4bfb850ad75dda7320702a5f1d77982/packages/%40react-spectrum/dnd/src/useDnDHooks.ts#L87-L90)

Reply:
Oh, maybe you can do something like this?

```js
useDraggableCollectionState({...props, ...options, getItems: props.getItems});
```

Reply:
Doesn't seem to work, the core issue is that `useDraggableCollectionState` will have `getItems` as a required prop due to the type, so components that access `dragAndDropHooks.useDraggableCollectionState` will be expected to provide it. However, typescript doesn't understand that `getItems` should only be provided via `useDragAndDrop`'s options and that if `getItems` isn't provided then the `useDraggableCollectionState` hook won't even be called in this specific ListView setup